@etobella How can one solve the failling tests?? More PR's tests seems to be failling.
But it is caused by another module which is not even included in the pr.

```
2022-05-06 13:19:33,922 287 ERROR odoo unittest.suite: ERROR: setUpClass (odoo.addons.edi_account_oca.tests.test_edi.EDIBackendTestCase)
Traceback (most recent call last):
  File "/__w/edi/edi/edi_account_oca/tests/test_edi.py", line 18, in setUpClass
    super().setUpClass(chart_template_ref=chart_template_ref)
  File "/opt/odoo/addons/account/tests/common.py", line 31, in setUpClass
    assert 'post_install' in cls.test_tags, 'This test requires a CoA to be installed, it should be tagged "post_install"'
AssertionError: This test requires a CoA to be installed, it should be tagged "post_install"
 
2022-05-06 13:19:34,209 287 INFO odoo odoo.modules.loading: Module edi_account_oca loaded in 0.99s (incl. 0.16s test), 135 queries (+2 test) 
2022-05-06 13:19:34,209 287 ERROR odoo odoo.modules.loading: Module edi_account_oca: 0 failures, 1 errors of 0 tests 

```

--- a/edi_oca/views/edi_exchange_record_views.xml
+++ b/edi_oca/views/edi_exchange_record_views.xml
@@ -221,6 +221,7 @@
                 <field name="exchanged_on" />
                 <field name="ack_received_on" />
                 <field name="edi_exchange_state" />
+                <field name="exchange_filename" />
                 <field name="identifier" />
                 <field name="external_identifier" />
                 <field name="ack_exchange_id" />

